libs/vre/resource-editor/representations/src/lib/resource-fetcher.service.ts

```diff
@@ -1,77 +1,43 @@
 import { Inject, Injectable } from '@angular/core';
-import { KnoraApiConnection, ReadResource, SystemPropertyDefinition } from '@dasch-swiss/dsp-js';
+import { KnoraApiConnection, SystemPropertyDefinition } from '@dasch-swiss/dsp-js';
 import { DspApiConnectionToken } from '@dasch-swiss/vre/core/config';
-import { OntologiesSelectors, SetCurrentResourceAction } from '@dasch-swiss/vre/core/state';
+import { SetCurrentResourceAction } from '@dasch-swiss/vre/core/state';
 import { DspResource, GenerateProperty } from '@dasch-swiss/vre/shared/app-common';
-import { ComponentCommunicationEventService, EmitEvent, Events } from '@dasch-swiss/vre/shared/app-helper-services';
-import { TranslateService } from '@ngx-translate/core';
 import { Store } from '@ngxs/store';
-import { BehaviorSubject, Subject, Subscription } from 'rxjs';
-import { map, switchMap, take } from 'rxjs/operators';
+import { combineLatest, Observable, ReplaySubject, Subject } from 'rxjs';
+import { map, startWith, switchMap } from 'rxjs/operators';
 
 @Injectable()
 export class ResourceFetcherService {
-  private _resourceIri!: string;
-  private _loadResourceSubject = new BehaviorSubject(null);
+  private _loadResourceSubject = new ReplaySubject<Observable<DspResource>>(1);
```

Comment:
Does replay an erased resource. When the fetcher onInits after erasing, it requests the resource and gets a 404 as the erased resource can not be requested anymore. Due to the errors the whole list and the component get stuck somehow and no navigation is possible anymore.